helm/hpcc/templates/thor.yaml

```diff
@@ -143,6 +145,7 @@ data:
             accessDali: "yes"
             accessEsp: "yes"
         spec:
+          {{- include "hpcc.placementMatchName" (dict "root" $ "pod" (printf "%s-" $eclAgentType) "group" "thor") | indent 10 }}
```

Comment:
as with other comments - I think should have .name in it, otherwise no way to apply different placements for this component to different thor's